<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherWindow</class>
 <widget class="QMainWindow" name="teacherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>361</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>学生信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>课程信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按课程登分</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>441</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnSubmit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRevert">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>撤销修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAdd">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>新增学生</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnDelete">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除学生</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAsend">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>220</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ID升序</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnDesend">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ID降序</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnFindByID">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>按学号查找</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnFindByName">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>按姓名查找</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditDom">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnFindByDom">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>330</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>按宿舍查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnBack">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>290</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>返回全表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>441</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnDeleteCourse">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除课程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnSubmit_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAddCourse">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>新增课程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRevert_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>撤销修改</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>441</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ComboCourse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnSubmit_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRevert_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>撤销修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnCal">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>计算成绩</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shaixuan">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>筛选</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>注销登录</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
